src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -32,12 +33,139 @@
       </ng-container>
     </cd-card>
   </div>
-
-  <div class="row mx-0">
+  <!-- Inventory Card -->
+  <div class="row mx-0 card-row">
     <cd-card title="Inventory"
              i18n-title
              class="col-sm-3 px-3">
-             Text
+      <ul class="inventory-list">
+        <!-- Hosts -->
+        <li>
+          <div *ngIf="healthData.hosts"
+               class="inventory-item d-flex justify-content-between">
+            {{ healthData.hosts }} Hosts
+            <span>
+              {{ healthData.hosts}}
+              <i class="fa fa-check-circle text-success"></i>
+            </span>
+          </div>
+        </li>
+        <hr />
+        <!-- Monitors -->
+        <li>
+          <div *ngIf="healthData.mon_status"
+               class="inventory-item d-flex justify-content-between">
+            {{ healthData.mon_status | monSummary }} Monitors
+            <span>
+              {{ healthData.mon_status | monSummary }}
+              <i class="fa fa-check-circle text-success"></i>
+            </span>
+          </div>
```

Comment:
You should try the inventory card with different Dashboard user roles: one that has only access to RGW stuff, another that can only manage RBD, and see what happens.

Summing up: we have to conditionally hide/display elements in the landing page based on:
* Feature toggle status,
* Permissions (@nSedrickm created a directive to simplify this work, and not to have to directly work with the auth storage service).

Reply:
You can check the current landing page to see how this is done.

Hoewever, for the roles I'd suggest that you use @nSedrickm's approach, and we should probably implement a similar one for the feature toggles, or even use the same one for both, since that directive basically tags HTML elements to Dashboard scopes, so if we ensure Auth scopes and Feature Toggles scopes are aligned, we get twice for the same price!